spec/features/deserialization_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.feature "deserialization" do
+  include ActiveJobHelper
+  include RackTestFeaturesHelper
+
+  before do
+    Timecop.travel Time.new(2015, 9, 1, 15, 47).utc
+    Capybara.current_driver = :rack_test
+    VisitMetricsEntry.delete_all
+  end
+
+  after do
+    Timecop.return
+  end
+
+  let(:data) {
+    YAML.load(
+      File.read(
+        File.expand_path('../../fixtures/deserialization_data.yml', __FILE__)
+      )
+    )
+  }
+
```

Comment:
As an old cat if mine would have said: "Hmph".  It burns my eyes, Monk.  It burns my eyes...

However, I also confess that I secretly like it and shall change styles, myself :)